README.md

```diff
@@ -12,20 +12,34 @@ This dockerized environment is designed for viewing private
 Clone the repo, navigate to the cloned directory and run the instance with:
 
 ```bash
-NODE_ENDPOINT=http://<node_endpoint> docker-compose up
+NODE_ENDPOINT=http://<node_endpoint> docker-compose -f <os>-docker-compose.yaml up
 ```
 Append the `-d` argument to run the containers in the backgroud
 
 You will be able to access the Explorer UI via:
 
 * http://localhost:5000
  
-### 7node example
+### Localhost
 
 You can set up the [7node](https://github.com/jpmorganchase/quorum-examples) example on your local machine and run the explorer with the following command.
```

Comment:
It would be useful to include a link to the relevant docker docs regarding docker.for.*.internal params.